java.source.base/test/unit/src/org/netbeans/api/java/source/TreePathHandleTest.java

```diff
@@ -422,6 +427,34 @@ public void testLocVar() throws Exception {
         assertNotNull(handle.resolveElement(info));
     }
 
+    public void testVarInstanceMember() throws Exception {
+        try {
+            SourceVersion.valueOf("RELEASE_10"); //NOI18N
+        } catch (IllegalArgumentException ex) {
+            //no RELEASE_10, skip test:
+            return;
+        }
+        FileObject file = FileUtil.createData(sourceRoot, "test/Test.java"); //NOI18N
+        String code = "package test; public class Test {var v;  \n public Test() {}}"; //NOI18N
+
+        writeIntoFile(file, code);
+        SourceUtilsTestUtil.setSourceLevel(file, "1.10"); //NOI18N
+        JavaSource js = JavaSource.forFileObject(file);
+
+        CompilationInfo info = SourceUtilsTestUtil.getCompilationInfo(js, Phase.RESOLVED);
+        assertTrue(info.getDiagnostics().size() > 0);
+
+        TreePath tp = info.getTreeUtilities().pathFor(code.indexOf("var") + 1); //NOI18N
+        VariableElement elem = (VariableElement) info.getTrees().getElement(tp);
+        try {
+            ClassFileUtil.createFieldDescriptor(elem);
+        } catch (IllegalArgumentException e) {
+            fail("IllegalArgumentException thrown"); //NOI18N
```

Comment:
unnecessary try-catch, if  IAE is thrown from a test method, JUnit reports a failure (error).

Reply:
Removed try-catch